scripts/test/mysql.sh

```diff
@@ -14,18 +14,18 @@
 
 set -e
 
-changed_fileext=$(git diff --name-only HEAD..develop|awk -F. '{print $NF}'|uniq)
+changed_fileext=$(git diff --name-only HEAD..origin/develop --|awk -F. '{print $NF}'|uniq)
 if [[ "$changed_fileext" == "md" ]]; then
     echo "Only Markdown files changed.  No need to run unit tests."
     exit 0
 fi
 
-# Wait for the creation of file /work/mysql-inited.  The entrypoint
-# of sqlflow:mysql should create this file on a bind mount of the host
-# filesystem.  So, the container running this script should also bind
-# mount the same host directory to /work.
-while read -r i; do if [ "$i" = "mysql-inited" ]; then break; fi; done \
-    < <(inotifywait  -e create,open --format '%f' --quiet /work --monitor)
+
+port=$(echo "$SQLFLOW_TEST_DB_MYSQL_ADDR" | awk -F":" '{print $2}')
+echo "Waiting mysql to launch on $port..."
+while ! nc -z localhost "$port"; do   
```

Comment:
I believed the MySQL container would write a file `mysql-inited` after loading the example datasets, so we can not do this change?
https://github.com/sql-machine-learning/sqlflow/blob/develop/docker/mysql/start.bash#L75

Reply:
We start the MySQL container using docker, so the file `mysql-inited` is owned by root user, which can not be read by a normal Linux user.